app/scripts/views/settings/clients.js

```diff
@@ -45,28 +49,45 @@ define(function (require, exports, module) {
       }
 
       var devices = this._devices;
-      devices.on('add', this._onDeviceAdded.bind(this));
-      devices.on('remove', this._onDeviceRemoved.bind(this));
+      devices.on('add', this._onItemAdded.bind(this));
+      devices.on('remove', this._onItemRemoved.bind(this));
+
+      if (! this._apps) {
+        this._apps = new Apps([], {
+          notifier: options.notifier
+        });
+      }
+
+      this._apps.on('add', this._onItemAdded.bind(this));
+      this._apps.on('remove', this._onItemRemoved.bind(this));
     },
 
-    _formatDevicesList: function (devices) {
-      return _.map(devices, function (device) {
-        if (device.lastAccessTimeFormatted) {
-          device.lastAccessTime = Strings.interpolate(
-            t('Last active: %(translatedTimeAgo)s'), { translatedTimeAgo: device.lastAccessTimeFormatted });
+    _formatAccessTime: function (items) {
+      return _.map(items, function (item) {
+        if (item.lastAccessTimeFormatted) {
+          item.lastAccessTimeFormatted = Strings.interpolate(
+            t('Last active: %(translatedTimeAgo)s'), { translatedTimeAgo: item.lastAccessTimeFormatted });
         } else {
-          // unknown lastAccessTime or not possible to format.
-          device.lastAccessTime = '';
+          // unknown lastAccessTimeFormatted or not possible to format.
+          item.lastAccessTimeFormatted = '';
         }
-        return device;
+        return item;
       });
     },
 
     context: function () {
+      // Create a combined collection to be able to sort apps and devices together.
+      var appsAndDevicesCollection = new AppsAndDevices();
```

Comment:
I'm confused by creating a new AppsAndDevices for just display, how come AppsAndDevices isn't the only model used?